ceremonies/2023/i1-cert.crl.yaml

```diff
@@ -0,0 +1,18 @@
+ceremony-type: crl
+pkcs11:
+  module: /usr/lib/x86_64-linux-gnu/softhsm/libsofthsm2.so
+  pin: 1234
+  signing-key-slot: 1307844626
+  signing-key-label: root-x2
+inputs:
+  issuer-certificate-path: ../2020/root-x2.cert.pem
+outputs:
+  crl-path: ./int-i1.crl.pem
+crl-profile:
+  this-update: 2023-09-25 00:00:00
+  next-update: 2024-08-04 00:00:00
+  number: 111
```

Comment:
Need to be 100% sure that this number remains higher than the current CRL number (103) when this ceremony is conducted.